lightning/src/util/events.rs

```diff
@@ -342,6 +342,13 @@ pub enum Event {
 	/// [`ChannelManager::claim_funds`]: crate::ln::channelmanager::ChannelManager::claim_funds
 	/// [`ChannelManager::fail_htlc_backwards`]: crate::ln::channelmanager::ChannelManager::fail_htlc_backwards
 	PaymentReceived {
+		/// The node that received the payment.
+		/// This is useful to identify payments which were received via [phantom node payments].
+		/// This field will always be filled in when the event was generated by LDK 
```

Comment:
Super-tiny nit: pushing `versions` to the next line here and below looks a bit off (and introduced trailing whitespaces again). Feel free to leave as is though, if no other changes are requested.

Reply:
hmm, so maybe I just keep it as one long line? not sure where else in that sentence to break on to keep it under the desired character limit

Reply:
Huh? If you'd keep "versions" on the previous line, my editor shows a line width of 92?

Reply:
I was trying to follow this guideline: https://github.com/lightningdevkit/rust-lightning/pull/1766#pullrequestreview-1186534248 but will update to move `versions` to the line below, no biggie